designer/client/cypress/e2e/components.cy.ts

```diff
@@ -196,7 +196,7 @@ describe("Components list", () => {
             .click();
         cy.wait(500); //ensure "loading" mask is hidden
         cy.get("#app-container>main").matchImage({
-            maxDiffThreshold: 0.01,
+            maxDiffThreshold: 0.02,
```

Comment:
I'm not sure about it..

Reply:
Yeah, I'm not a fan of that either - without it tests cant complete (because scenarios uses timestamp in their names). I've started conversation about it elsewhere - hope to get it resolved soon.

Reply:
If there is a possibility like that and name does not matter in any way in that test then it can be an option. I was rather thinking about creating a name in more predictable way so it wont change on every test.